meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -32,6 +32,7 @@ IMAGE_INSTALL_append = " \
     packagegroup-gdp-qt5 \
     packagegroup-gdp-rvi \
     packagegroup-gdp-dev \
+    chromium-wayland \
```

Comment:
@zeenix The HMI button is added by the chromium-wayland recipe and it will only run if you add Chromium to the image, so that won't be a problem. If chromium is not in the image, the button will not be added.

@chbae: oh, that looks cleaner, indeed. Let me change it.